releasenotes/notes/change-context-relevance-evaluator-da770bf05c83cfbd.yaml

```diff
@@ -0,0 +1,6 @@
+---
+
+features:
```

Comment:
This is a breaking change.

Reply:
what's the correct [keyword/section](https://github.com/deepset-ai/haystack/blob/main/releasenotes/config.yaml) to describe a breaking change? `features` or `issues` or `upgrade`?

Reply:
I've added as `upgrade`